aws/README.md

```diff
@@ -358,6 +358,107 @@ data='
+
+#### Generate the data
+
+1. `git clone git://github.com/gpii/universal.git && cd universal && npm install`
+1. Put the preferences set you want to load into a folder, i.e.: _testData/myPrefsSets_.
+1. Create a folder to store the db data, i.e.: _testData/myDbData_.
+1. `node ./scripts/convertPrefs.js testData/prefSets testData/myDbData`
+
+Now there are two new files inside _testData/myDbData_, _gpiiKeys.json_ and _prefsSafes.json_.
+Both files contain an array of JSON objects, i.e.:
+
+```
+[
+    {
+        "_id": "GPII-270-rbmm-demo",
+        "type": "gpiiKey",
+        "schemaVersion": "0.1",
+        "prefsSafeId": "prefsSafe-GPII-270-rbmm-demo",
+        "prefsSetId": "gpii-default",
+        "revoked": false,
+        "revokedReason": null,
+        "timestampCreated": "2018-08-12T16:34:04.656Z",
+        "timestampUpdated": null
+    },
+    {
+        "_id": "MikelVargas",
+        "type": "gpiiKey",
+        "schemaVersion": "0.1",
+        "prefsSafeId": "prefsSafe-MikelVargas",
+        "prefsSetId": "gpii-default",
+        "revoked": false,
+        "revokedReason": null,
+        "timestampCreated": "2018-08-12T16:34:04.661Z",
+        "timestampUpdated": null
+    }
+]
+```
+
+And we want to edit these files to look like the following:
+
+```
+{ "docs":
+    [
+        {
+            "_id": "GPII-270-rbmm-demo",
+            "type": "gpiiKey",
+            "schemaVersion": "0.1",
+            "prefsSafeId": "prefsSafe-GPII-270-rbmm-demo",
+            "prefsSetId": "gpii-default",
+            "revoked": false,
+            "revokedReason": null,
+            "timestampCreated": "2018-08-12T16:34:04.656Z",
+            "timestampUpdated": null
+        },
+        {
+            "_id": "MikelVargas",
+            "type": "gpiiKey",
+            "schemaVersion": "0.1",
+            "prefsSafeId": "prefsSafe-MikelVargas",
+            "prefsSetId": "gpii-default",
+            "revoked": false,
+            "revokedReason": null,
+            "timestampCreated": "2018-08-12T16:34:04.661Z",
+            "timestampUpdated": null
+        }
+    ]
+}
+
+```
+
+For the lazy:
+* `cd testData/myDbData`
+* `sed -i '1i { "docs": ' prefsSafes.json gpiiKeys.json`
+* `sed -i '$ a }' prefsSafes.json gpiiKeys.json`
+
+Congratulations, half of the work is done, now on to the load step.
+
+#### Load the data into a cluster
+
+Requirements:
+* You are either an Op or you have been trained by one of them in using gpii-infra.
+* Your computer is already set up to work with gpii-infra.
+* And the most important thing, you know what you are doing. If not, ask the Ops team ;)
+
+We assume that you are going to perform this steps into an already up & running cluster.
+
+1. Go into the corresponding folder with the cluster name where you want to perform the process, "stg", "prd" or "dev". In this case We are going to perform the process in "dev": `cd gpii-infra/aws/dev`
```

Comment:
The recommended workflow is:
1. Test the changes in your dev cluster
1. Make the changes in stg if everything worked as expected in the previous step
1. Make the changes in prd if everything worked as expected in the previous step
:wink: 

Perhaps a link to the list of the environments with the order could also help:
https://github.com/gpii-ops/gpii-infra/blob/1c8d7680c5fb2aa4dcb8e6f1f1b52282210cd91a/aws/README.md#what-are-all-these-environments